video/out/vulkan/context_mac.m

```diff
@@ -50,6 +50,11 @@ static void mac_vk_get_vsync(struct ra_ctx *ctx, struct vo_vsync_info *info)
     [p->vo_mac fillVsyncWithInfo:info];
 }
 
+static int mac_vk_color_depth(struct ra_ctx *ctx)
+{
+    return 10;
```

Comment:
Can we at least check backbuffer type before returning this? This will be wrong if we have swapchain configured to ARGB8888

Reply:
there is probably some way. though i am not sure if this would conflict with how moltenvk works, since we would need to retrieve the current drawable most likely, which moltenvk would to too for drawing.

Reply:
irrelevant now, though maybe useful in the future or just as an info. it's easier than i though, if ever needed we can just check the `pixelFormat` property on the metal layer https://developer.apple.com/documentation/quartzcore/cametallayer/1478155-pixelformat